types/cache-interceptor.d.ts

```diff
@@ -78,6 +78,7 @@ declare namespace CacheHandler {
      * @default Infinity
      */
     maxEntrySize?: number
+    errorCallback?: (err: unknown) => void
```

Comment:
Let's help callers assume its an error
```suggestion
    errorCallback?: (err: Error) => void
```